sdk/iot/provisioning/src/az_iot_provisioning_client.c

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
```

Comment:
General comment: 
please make all the const in the code are some const at the top of file define static like you have done in line 15 for literal "&ClientVersion="

Reply:
I actually now think that having the consts at the top makes the code harder to read. I will move most of these within the functions unless they are used in two places.